File: fizz-buzz/fizz-buzz.go
func fizzBuzz(n int) []string {
	// Solutin using a switch
    var result []string
	for i := 1; i <= n; i++ {
		switch {
		case i%5 == 0 && i%3 == 0:
			result = append(result, "FizzBuzz")
		case i%3 == 0:
			result = append(result, "Fizz")
		case i%5 == 0:
			result = append(result, "Buzz")
		default:
			result = append(result, strconv.Itoa(i))
		}
	}
	return result
}

File: fizz-buzz/fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        result = []
        for i in range(n):
            num = i+1
            if (num % 5 == 0) and (num %3 == 0):
                result.append("FizzBuzz")
            elif num % 3 == 0:
                result.append("Fizz")
            elif num % 5 == 0:
                result.append("Buzz")
            else:
                result.append(str(num))
        return result